src/courtroom.cpp

```diff
@@ -242,6 +242,8 @@ Courtroom::Courtroom(AOApplication *p_ao_app) : QMainWindow()
   ui_pre_non_interrupt->hide();
 
   ui_custom_objection = new AOButton(this, ao_app);
+  ui_custom_objection->setContextMenuPolicy(Qt::CustomContextMenu);
+  custom_obj_menu = new QMenu;
```

Comment:
Is this really supposed to have a `nullptr` parent?

Reply:
@cents02

Reply:
@oldmud0 where do you see the null pointer

Reply:
The default constructor of a QWidget sets the parent to `nullptr` so it doesn't get garbo collected when everything else is.

Reply:
@oldmud0 I see, since custom objection list is static and is used as long as the courtroom exists I don't see why it should be collected. It's not as if it will take much memory but yeah I agree that better practise would be to destroy it along with the courtroom

Reply:
So:
```suggestion
  custom_obj_menu = new QMenu(this);
```

Reply:
@in1tiate since this is your pr and I don't want to step on anything please do oldmud0's suggestion

Reply:
@cents02 Got it, thanks for taking a look at this for me